server-base/src/main/java/org/apache/kylin/rest/request/SQLRequest.java

```diff
@@ -105,7 +105,7 @@ public Object getCacheKey() {
             return cacheKey;
 
         cacheKey = Lists.newArrayList(sql.replaceAll("[ ]", " ") //
-                , project //
+                , getNormProject() //
```

Comment:
Can we normalize the value to upper case when the request is received? Then downstream doesn't need to handle the case each time.